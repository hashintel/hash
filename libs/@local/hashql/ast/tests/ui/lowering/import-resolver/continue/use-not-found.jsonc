//@ run: pass
//@ description: Tests that a `use` statement for a non-existent module (`kernel::foo`) is handled by dummy replacement, allowing lowering to continue.
[
  "use",
  "kernel::foo",
  //~^ ERROR Module 'foo' not found
  "*",
  ["add", { "#literal": 1 }, { "#literal": 2 }]
]
