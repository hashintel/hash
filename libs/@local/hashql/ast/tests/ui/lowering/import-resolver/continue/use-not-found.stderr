[import-resolver::unresolved-import] Error: Unresolved import
   ╭─[ <unknown>:5:12 ]
   │
 5 │   "kernel::foo",
   │    ─────┬───┬─  
   │         │   ╰─── Module 'foo' not found
   │         │       
   │         ╰─────── In this path
   │ 
   │ Help: The module 'kernel::foo' doesn't exist in this scope. Check the spelling and ensure the module is available.
   │ 
   │ Note: Modules must be properly defined and exported from their parent module to be accessible.
───╯
